### PR TITLE
Dynamic Views and Styling

### PR DESCRIPTION
**Pull request by migo1**
### MILESTONES

- Use preprocessed HTML file with embedded Ruby code.
- Use layouts and templates for shared content.

**please run the seeders**

### CODE REVIEW

File: app/views/users/show.html.erb [hunk 1]
```diff
@@ -1,11 +1,29 @@
...
+        <p> <%= post.title %></p>
+        <p class="mb-2"><%= post.text %></p>
+        <div class="flex w-full flex-row-reverse">
+          <p>Likes: <%= post.likes_counter %></p>
+          <p class="mr-2">Comments: <%= post.comments_counter %></p>
+        </div>
+      </a>
+    <% end %>
+    <button class="border-2 border-black w-28 mx-auto mt-2">see all posts</button>
+  </div>
```
Comment (tresorsawasawa) [inline app/views/users/show.html.erb]: - [x] As required for the project, kindly ensure that when clicking on the **See all Post** link/button it opens `user's post index` page(i.e `app/views/posts/index.html.erb`) as displayed on the [wireframe](https://github.com/microverseinc/curriculum-rails/raw/main/blog-app/images/blog_user_all_posts.png)

**Review by tresorsawasawa — CHANGES_REQUESTED** [review]
# STATUS: REQUIRED CHANGES :recycle::recycle: 

Hi @migo1 
Good job so far! I know you worked hard to implement this project, I really appreciate the effort you put in! But there are some issues that you still need to work on to go to the next project but you are almost there! 

almost-there

## To highlight :champagne: 
- All views are renering :heavy_check_mark: 
- Used embeded ruby :heavy_check_mark: 

## Required changes :recycle: :recycle: 
- [ ] Check the comments under the review.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

_Feel free to leave any questions or comments in the PR thread if something is not 100% clear. 
Please ping me [@tresorsawasawa](https://github.com/tresorsawasawa) when you comment so I can receive the notification or use slack with the same name._

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: app/views/users/show.html.erb (hunk 1 above)
Comment (migo1) [reply app/views/users/show.html.erb]: noted

**Review by fobadara — APPROVED** [review]
Hi @migo1, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

APPRVED
## Highlights
✅ Descriptive README
✅ Descriptive PR summary
✅ Clean code
✅ Good use of separation of concerns


## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**